### DISCUSSION TITLE
Schemaless Attributes modelScope() method with orWhere

### DISCUSSION THREAD

**Discussion by chatisk** [spatie/laravel-schemaless-attributes#104] [Q&A]
I would like to ask if there what i am asking below it is implemented already (which I am missing clearly) or need to implement it.

In modelScope() method in src>SchemalessAttributes, we have the modelScope() method, which builds the query to search a value in the attributes.

I wanted to chain multiple WHEREs , instead of making a separate query for each attribute  (and then combining the results) , i wanted to do.
`
foreach ($this->allResourceAttributes as $attribute) {
      $modelAttributeQuery->withExtraAttributes($attribute, 'LIKE', '%' . $this->search . '%');
 }
`

Deep down in the modelScope i see that it says
`
$builder->where("{$this->sourceAttributeName}->{$name}", $operator ?? '=', $value)
`

This is going to do a `SELECT * from Model WHERE attribute LIKE value AND attribute LIKE value AND etc...` (simplified the statement here , there is much json work here)

My goal is achieved is achieved if I switch the `where` to `orWhere`  . I wanted to ask , is the package implementing this orWhere somehow else ? 

Thank you in advance for your time to respond , much appreciated!

**Comment by MCKLtech** [comment]
Did you ever solve this? I think it would be helpful for solving my issue too: https://github.com/spatie/laravel-schemaless-attributes/discussions/112

**Reply by chatisk** [reply]
@MCKLtech the PR is here , i never had the time to delve into the tests as requested https://github.com/spatie/laravel-schemaless-attributes/pull/109

**Reply by MCKLtech** [reply]
Thank you, appreciate it.
